test/support/api/program-actions.js

```diff
@@ -53,7 +53,7 @@ Cypress.Commands.add('routeAllProgramActions', (mutator = _.identity, programIds
     .fixture('test/roles').as('fxRoles');
 
   cy.route({
-    url: '/api/program-actions?*',
```

Comment:
this is the only change that makes me nervous as it essentially matches the same route as a single program-action.  Is it not matching with the `?` it seems like any use of this would call `?filter[status]`   Otherwise maybe no `*`?

Reply:
looks like it works fine if I switch it back....must have initially switched it when something else was wrong?